bugsnag/event.py

```diff
@@ -124,6 +124,9 @@ def get_config(key):
         for name, tab in options.items():
             self.add_tab(name, tab)
 
+        if hasattr(exception, "__notes__"):
+            self.add_tab("exception notes", dict(enumerate(exception.__notes__)))
```

Comment:
This fails linting because the line is every so slightly too long

> bugsnag/event.py:128:80: E501 line too long (81 > 79 characters)
>
> https://github.com/bugsnag/bugsnag-python/actions/runs/5452725832/jobs/9920702076?pr=340#step:5:71

```suggestion
            self.add_tab(
                "exception notes",
                dict(enumerate(exception.__notes__))
            )
```